docs/build.sbt

```diff
@@ -17,13 +17,13 @@ lazy val docs = project
   .settings(forkedTests: _*)
   .settings(
     resolvers += Resolver.typesafeIvyRepo("releases"),
-    scalaVersion := "2.11.7",
+    scalaVersion := "2.11.11",
     libraryDependencies ++= Seq(
       "com.typesafe.akka" %% "akka-stream-testkit" % AkkaVersion % "test",
       "org.apache.cassandra" % "cassandra-all" % "3.0.9" % "test",
       "junit" % "junit" % "4.12" % "test",
       "com.novocode" % "junit-interface" % "0.11" % "test",
-      "org.scalatest" %% "scalatest" % "3.0.1" % Test,
+      "org.scalatest" %% "scalatest" % "3.0.3" % Test,
```

Comment:
Update here for the docs to keep consistent with the version used by the main project.